api/src/log.rs

```diff
@@ -17,7 +17,7 @@ use std::time::Duration;
 
 const REQUEST_SOURCE_CLIENT_UNKNOWN: &str = "unknown";
 static REQUEST_SOURCE_CLIENT_REGEX: Lazy<Regex> =
-    Lazy::new(|| Regex::new(r"aptos-[a-zA-Z]+/[0-9A-Za-z\.\-]+").unwrap());
+    Lazy::new(|| Regex::new(r"aptos-[a-zA-Z\-]+/[0-9A-Za-z\.\-]+").unwrap());
```

Comment:
does `\-` mean "any amount of `-`"?

Reply:
`[a-zA-Z\-]+` means "at least one of these chars", where "these chars" is a-z, A-Z, and -.

Reply:
I wonder if we should also consider digits. not important for now but who knows what names others would come up with. we ask for the format `<sdk-id>/<sdk-version>` where `<sdk-id>` can be anything